TA_dataset/bin/dataset_search_command.py

```diff
@@ -116,17 +118,19 @@ def get_search_arguments(self):
     )
 
 
-def search_error_exit(self, r_json):
+def search_error_exit(self, r_json: Union[Dict[str, Any], str]) -> None:
```

Comment:
Do we have any unit tests for this function?

Reply:
I have added the unit test.